package.json

```diff
@@ -4,9 +4,8 @@
   "description": "Setting up a project with web-pack",
   "private": true,
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "webpack serve --open",
-    "build": "webpack"
+    "start": "webpack serve",
```

Comment:
- [ ] OPTIONAL
The _script option_ of your _package.json_ file is **not properly configured** and hence your application does not launch in a browser when the command `npm start` is run. Kindly change the _script options_  `"start": "webpack serve",` **to** `"start": "webpack serve --open"`, in your _packge.json_ file.

**Your script option in package.json file:**
![image](https://user-images.githubusercontent.com/12745474/135311341-4875e706-b201-4bd6-8030-287c391d041f.png)

Reply:
Thank you for the approval, brother. Let me fix it